### DISCUSSION TITLE
additional runner metrics

### DISCUSSION THREAD

**Discussion by nicholasgibson2** [actions/actions-runner-controller#1399] [Questions] [answered]
It would very useful to have **PercentageRunnersBusy** and **TotalNumberOfQueuedAndInProgressWorkflowRuns** available in the metrics service. It would also be useful having the **active/idle status** per runner. Any thoughts on how difficult this would be to implement?

**Comment by mumoshu** [comment, marked as answer]
@nicholasgibson2 Hey! I tend to recommend https://github.com/kaidotdev/github-actions-exporter for job-related metrics and https://github.com/erikkn/github-actions-exporter for runner-related metrics. The former should be enough to calculate TotalNumberOfQueuedAndInProgressWorkflowRuns and the latter for PercentageRunnersBusy.

**Reply by mumoshu** [reply]
For more context, see https://github.com/actions-runner-controller/actions-runner-controller/pull/820
